### DISCUSSION TITLE
Sveltekit Embed on mobile devices

### DISCUSSION THREAD

**Discussion by Amplitude88** [spences10/sveltekit-embed#352] [Q&A]
Hello,

have anybody a sveltekit-embed app which is working fine on mobile devices? I have the problem, if i use mobile devices the Observer isn't load the script/iframe.
On Desktop its working fine, but on mobile device it isn't working properly. 

<GenericEmbed 
width = '100%'
height = "700px"
src='https://calendly.com/metaempirex/strategy-call?background_color=0d0b27&text_color=ffffff&primary_color=6e03cb'
allowtransparency="true" 
style="background: #000000;"
/>

Thanks for help

**Comment by spences10** [comment]
Hey, do you know what mobile devices?

It's working ok for me on iOS (Safari) and Android (Google Chrome)
